formats/iso0.go

```diff
@@ -39,6 +72,25 @@ func (i *ISO0) Encode(pin, account string) (string, error) {
 		return "", err
 	}
 
+	// write encode information
+	{
+		tw := i.getWriter()
+		fmt.Fprintf(tw, "PIN block encode operation finished\n")
+		fmt.Fprintf(tw, "%s\n", strings.Repeat("*", 36))
+		fmt.Fprintf(tw, "PAN\t: %s\n", account)
+		fmt.Fprintf(tw, "PIN\t: %s\n", pin)
+		if pad, _ := i.padding(pin); len(pad) == 0 {
+			fmt.Fprintf(tw, "PAD\t: N/A\n")
+		} else {
+			fmt.Fprintf(tw, "PAD\t: %s\n", pad)
+		}
+		fmt.Fprintf(tw, "Format\t: %s\n", i.format())
+		fmt.Fprintf(tw, "%s\n", strings.Repeat("-", 36))
+		fmt.Fprintf(tw, "Encoded PIN block\t: %s\n", strings.ToUpper(xorBlock))
+		fmt.Fprintf(tw, "Encoded PAN block\t: %s\n", strings.ToUpper(accountBlock))
```

Comment:
I'm not sure about `Encoded`. Maybe `Formatted` is better here? I think that result is - encoded, but the rest is... not sure.